app/controllers/feedback_messages_controller.rb

```diff
@@ -8,8 +8,7 @@ def create
 
     params = feedback_message_params.merge(reporter_id: current_user&.id)
     @feedback_message = FeedbackMessage.new(params)
-
-    if (recaptcha_disabled? || recaptcha_verified?) && @feedback_message.save
+    if (recaptcha_disabled? || recaptcha_verified? || params[:feedback_type] == "connect") && @feedback_message.save
```

Comment:
Maybe you want to wrap up `params[:feedback_type] == "connect"` in a new predicate method too?

```suggestion
    if (recaptcha_disabled? || recaptcha_verified? || connect_feedback?) && @feedback_message.save
```